zigen-project/zen
# Connect to the headsets from UI

## Summary
Added a button in menu 2D destktop so that the users can start the HMD mode with a button

## How to check behavior
- Click the "VR" button in the right-bottom corner and confirm that all the available headsets are listed.
- Click one of the "Connect" button and confirm that the action leads you to the HMD mode.
